### PR TITLE
Rename perfetto_compiled to perfetto_ui_compiled to match g3 config

### PR DESCRIPTION
**Pull request by kenzieschmoll**
No functional changes - just a rename.

RELEASE_NOTE_EXCEPTION=[not user facing]

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -29,3 +29,6 @@ packages/devtools_test/lib/src/mocks/generated.mocks.dart
 **/test/**/failures/
 *.dill
 DEPENDENCIES.md
+
+# Ignore logs generated by VS Code
+**/logs/**
```
Comment (elliette) [inline .gitignore:34]: Is this change intentional?

**Review by elliette — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (kenzieschmoll) [reply .gitignore:34]: Yeah I can break it out to a separate PR if we want, but I accidentally committed some logs that VS code created in my first commit, so just figured I'd add it while I was here.
